src/runtime/runtime_unix.go

```diff
@@ -256,3 +256,41 @@ func growHeap() bool {
 	setHeapEnd(heapStart + heapSize)
 	return true
 }
+
+func init() {
+	// Set up a channel to receive signals into.
+	// A channel size of 1 should be sufficient in most cases, but using 4 just
+	// to be sure.
+	signalChan = make(chan uint32, 4)
+}
+
+var signalChan chan uint32
+
+//go:linkname signal_enable os/signal.signal_enable
+func signal_enable(s uint32) {
```

Comment:
I think it is a good idea to add the remaining signal functions (`signal_disable`, `signal_ignore`, `signal_ignored`) golang provides as well, or at least stub them out. I remember building applications relying on the signal packages and often requiring most of them.